common/src/main/java/org/tron/common/utils/ByteUtil.java

```diff
@@ -342,7 +342,7 @@ public static int numberOfLeadingZeros(byte[] bytes) {
   public static byte[] parseBytes(byte[] input, int offset, int len) {
 
     if (offset >= input.length || len == 0) {
-      return EMPTY_BYTE_ARRAY;
+      return new byte[len];
```

Comment:
Yes, I missed it. I initially thought to place this fix in the `ByteUtil.parseWord` and it shouldn't affect anything since it is used only in BN128 precompiles. But `parseBytes` is used in other places too so it isn't correct.

Reply:
Thank you, I'll fix it